### ISSUE TITLE
Save failed

### ISSUE THREAD

**Issue by nuthankumarns** [edge-reflow/issues#122]
After creating couple of media queries and designing, reflow failed to save and the reflow title always shows Untitled*

**Comment by jtlafferty** [comment]
This has happened on some occasions in the past. Can you mention what version you are on? 

If you restart Reflow, does Save begin working correctly?

**Comment by anirudhsasikumar** [comment]
User on twitter reported the same bug after our latest release. Dev team is actively trying to reproduce the probem - any steps/clues anyone may have will be invaluable.

**Comment by nuthankumarns** [comment]
Adobe Edge Reflow-CC Version 0.32.13658(preview)
Regrets, I cannot give much debug info of the point during the freeze.

**Comment by anirudhsasikumar** [comment]
Some more data from in-app feedback: BUG - application wont save periodically despite changes being made. Won't prompt to save when closing app either, so work was lost. WORKAROUND - dragging an image from assets on to the workspace, then deleting it will put the app in a state where saving is possible.

**Comment by jtlafferty** [comment]
@nuthansantharam, did you start in PS and import into Reflow? I'm just trying to find some patterns as to when this save issue happens.

Also, what OS version do you have running?

Thanks.

**Comment by nuthankumarns** [comment]
Yes.. psd to html import and editing/modifying did not save the project.

On Thu, Jan 23, 2014 at 5:28 AM, Joan Lafferty notifications@github.comwrote:

> @nuthansantharam https://github.com/nuthansantharam, did you start in
> PS and import into Reflow? I'm just trying to find some patterns as to when
> this save issue happens.
> 
> Thanks.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/edge-reflow/issues/issues/122#issuecomment-33083450
> .

## 

Thanks,
Nuthan Santharam.

**Comment by AllieTheDemonHunter** [comment]
After importing from PS to Rf and setting up layouts for numerous break-points save (repeated) did not work. After closing application and re-launching; the saved file contains no work that was committed.

Rf version: 0.37.15833
OS version: OS X 10.9.1

Any help appreciated..

**Comment by nimbupani** [comment]
@nuthansantharam @AllieTheDemonHunter could you by any chance send your PSD files to me to investigate this issue? manian@adobe.com Thanks!

**Comment by nimbupani** [comment]
Thank you so much! I am able to reproduce this.

**Comment by nimbupani** [comment]
@AllieTheDemonHunter @nuthansantharam you can temporarily try to use 'Save As' and it should work.
